src/App.tsx

```diff
@@ -21,14 +71,19 @@ export const App: React.FC = () => {
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {isLoading && <Loader />}
+              {!isError && !isLoading && <TodoList todos={visibleTodos} />}
+              {isError && !isLoading && (
```

Comment:
it looks so complex, try to split this logic using the ternary operator
```
{isLoading ? <Loader /> : ...}
```

Reply:
@etojeDenys I could do it when I used Todo[] | null. But, after removing option 'null' I can't do it anymore